packages/tofu/test/inspectEsmImports.spec.js

```diff
@@ -29,58 +26,40 @@ testInspect(
   import defaultExport3, * as name2 from "package09";
   import "package10";
   var promise = import("package11");
+  export * as name3 from "package12";
+  export { name4 } from "package13";
+  export { default as name5 } from "package14";
+  export const foo = "bar";
 `,
-  {
-    esmImports: [
-      'package01',
-      'package02',
-      'package03.export1',
-      'package04.export2',
-      'package05.export3',
-      'package05.export4',
-      'package06/path/to/specific/un-exported/file.export5',
-      'package06/path/to/specific/un-exported/file.export6',
-      'package07.export7',
-      'package07.export8',
-      'package08',
-      'package09',
-      'package09',
-    ],
-  }
+  [
+    'package01',
+    'package02',
+    'package03',
+    'package04',
+    'package05',
+    'package06/path/to/specific/un-exported/file',
+    'package07',
+    'package08',
+    'package09',
+    'package10',
+    'package12',
+    'package13',
+    'package14',
+  ]
```

Comment:
This doesn't seem equivalent. Are we dropping information that's actually unnecessary?

Reply:
It's not equivalent with the previous behavior.  We're dropping the granularity down to module level.  Previously it was granular to named exports.

Reply:
sounds like a big improvement.